src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -256,13 +256,7 @@ function MoneyRequestConfirmationList({
 
     const isMovingTransactionFromTrackExpense = IOUUtils.isMovingTransactionFromTrackExpense(action);
 
-    const distance = useMemo(() => {
-        const value = TransactionUtils.getDistance(transaction);
-        if (canUseP2PDistanceRequests && isMovingTransactionFromTrackExpense && unit && !TransactionUtils.isFetchingWaypointsFromServer(transaction)) {
-            return DistanceRequestUtils.convertToDistanceInMeters(value, unit);
-        }
-        return value;
-    }, [isMovingTransactionFromTrackExpense, unit, transaction, canUseP2PDistanceRequests]);
```

Comment:
The distance logic here was updated in https://github.com/Expensify/App/pull/43819 to fix https://github.com/Expensify/App/issues/42959. The issue they face is actually the same as ours, which is the `transaction.comment.customUnit.quantity` value is in meters from GetRouteForDraft, but in the workspace unit when the request is completely created.

I have retested the step from the linked PR and verify it still works as expected as in the linked PR.

https://github.com/user-attachments/assets/90c4e076-f3c5-4af6-9791-e3052fa9a6d4